content/docs/01-announcing-ai-sdk-5-alpha/index.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: AI SDK 5 Alpha
+description: Get started with the Alpha version of AI SDK 5.
+---
+
+# Announcing AI SDK 5 Alpha
+
+<Note type="warning">
```

Comment:
Perhaps we should put a date on this so folks landing on it now have some sense of when we started the early preview